docs/development-environment.md

```diff
@@ -0,0 +1,167 @@
+# Development Environment
+
+## Table of contents
+
+* [Setting up your environment](#setting-up-your-environment)
+	* [Installing development tools](#installing-development-tools)
+* [Start development](#development-workflow)
+	 * [Run a build](#building-go)
+* [Unit Testing](#unit-testing)
+	 * [PHP unit testing](#php-unit-tests)
+* [Linting, standards, etc.](#linting-standards-etc)
+	* [Coding standards](#coding-standards)
+	* [Linting](#linting)
+
+# Setting up your development environment
+
+## Define `SCRIPT_DEBUG`
+
+When developing locally with Go, you should set the `SCRIPT_DEBUG` constant inside of your `wp-config.php` file to `true`. This ensures that all of the unminified files are loaded, since none of the minified js/css files exist in the theme until you run a final production build.
+
+```php
+define( 'SCRIPT_DEBUG', true );
+```
+
+## Installing development tools
+
+Scripts and styles are compiled using Webpack and a number of node packages. These can be quickly and easily installed using `npm`.
+
+To install all of the Go dependencies, you can run the two commands:
+
+```sh
+$ npm install
+$ composer install
+```
+
+If you are introducing or altering any styles, you'll most likely want to setup
+
+`.dev/config/webpack.settings.js`
+
+# Development workflow
+
+To start work on the Go theme you need to follow these steps:
+
+1. [Clone the repository](#clone-the-repository)
+2. [Install the development tools](#installing-development-tools)
+3. Make sure Go is enabled on your WordPress site
+4. [Build Go](#building-go)
+
+## Clone the repository
+
+Make sure you have `git`, `node`, `composer`, and a working WordPress installation.
+Clone this repository inside your local install theme directory.
+
+```sh
+git@github.com:godaddy-wordpress/go.git
+cd go
+```
+
+ You'll need to have a public SSH key setup with GitHub, which is more secure than saving your password in your keychain.
+ There are more details about [setting up a public key on GitHub.com](https://help.github.com/en/articles/adding-a-new-ssh-key-to-your-github-account).
+
+## Building Go
```

Comment:
Should we mention `npm run package` at all you think? Perhaps if they were building a zip to upload somewhere (for whatever reason)